qiita_db/data.py

```diff
@@ -1303,12 +1303,54 @@ def data_type(self, ret_id=False):
         return data_type[0]
 
     @property
-    def processed_date(self):
-        """Return the processed date"""
+    def processing_info(self):
+        """Return the processing item and settings used to create the data
+
+        Returns
+        -------
+        dict
+            Parameter settings keyed to the parameter, along with date and
+            algorithm used
+        """
+        # Get processed date and the info for the dynamic table
         conn_handler = SQLConnectionHandler()
-        return conn_handler.execute_fetchone(
-            "SELECT processed_date FROM qiita.{0} WHERE "
-            "processed_data_id=%s".format(self._table), (self.id,))[0]
+        sql = """SELECT processed_date, processed_params_table,
+            processed_params_id FROM qiita.{0}
+            WHERE processed_data_id=%s""".format(self._table)
+        static_info = conn_handler.execute_fetchone(sql, (self.id,))
+
+        # Get the info from the dynamic table, including reference used
+        sql = """SELECT * from qiita.{0}
```

Comment:
I'm getting confused in this SQL query, it looks like it is enough to do:

```sql
SELECT * FROM qiita.{0}
  JOIN qiita.reference USING (reference_id)
WHERE processed_params_id = {1}
```

I think this SQL query can return _only_ a single value, since there will be only one row with the given processed_params_id.